R/viz.R

```diff
@@ -4220,57 +4332,75 @@ fancy_circos_plot = function(this_sample,
     n_track = 4
   }
 
-  #get variants
-  svs = get_combined_sv(sample_ids = this_sample, projection = projection)
+  if(manta_calls){
```

Comment:
This probably can be updated, since the argument name implies manta, but the svs are from manta and GRIDSS? Maybe `use_sv_calls`?

Reply:
Good call!